### PR TITLE
Use uploaded artefact

### PR DESCRIPTION
**Pull request by grahamlyons**
The deployment should use the files bundled in the release archive which is uploaded to S3.

This obtains the archive from S3 and runs Terraform against it.

### CODE REVIEW

File: cdflow_commands/account.py [hunk 1]
```diff
@@ -8,28 +11,50 @@ def __init__(self, alias, id, role):
...
+        default_region, environment_mapping
     ):
         self.accounts = accounts
         self.release_account = release_account
         self.release_bucket = release_bucket
         self.default_region = default_region
+        self._environment_mapping = environment_mapping
 
-    def create(raw_scheme):
+    @classmethod
```
Comment (bart613) [inline cdflow_commands/account.py]: I had to look up what this does, never seen this before...

File: cdflow_commands/deploy.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    def run(self, plan_only=False):
+        self._plan()
+        if not plan_only:
+            self._apply()
+
+    def _plan(self):
+        with NamedTemporaryFile() as secrets_file_path:
+            json.dump(get_secrets(), secrets_file_path)
+            check_call(
+                self._build_parameters('plan', secrets_file_path),
```
Comment (bart613) [inline cdflow_commands/deploy.py]: Do we need to inject env variables here, too? (i.e. the ones in `./config/`)

File: cdflow_commands/account.py (hunk 1 above)
Comment (grahamlyons) [inline cdflow_commands/account.py]: Yeah, I preferred it over hardcoding `AccoutScheme.some_property` in the method.

File: cdflow_commands/deploy.py (hunk 2 above)
Comment (grahamlyons) [inline cdflow_commands/deploy.py]: No, that's bundled into the release: https://github.com/mergermarket/cdflow-commands/blob/c1cac077f111c4dff96aa05e02f43dce64b3efd0/cdflow_commands/release.py#L83

I didn't want to bundle decrypted config values though.

**Comment by tomyan** [comment]
Can we also remove `Metadata` and `load\_service\_metadata` from the config module?

**Pull request merged**